src/_modules/common_values/outputs_network.tf

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Do you think private dns zone are valuable?

Reply:
I see you introduced DNS zones but not the private ones which are useful for private endpoints

Reply:
sorry, I misread the comment, I added them now